### ISSUE TITLE
Channel - Trade Channel

### ISSUE THREAD

**Issue by Naraku93** [cmangos/tbc-db#946]
the trade channel is available in all zones, it should be available only in citys

**Comment by Cnzjs** [comment]
confirm,this should be a bug.

**Comment by Lhalyna** [comment]
this is right, the trade channel is only avaible in citys. i tested it on the official tbc server

**Comment by Cnzjs** [comment]
> this is right, the trade channel is only avaible in citys. i tested it on the official tbc server

but in cmangos now,this channel is available in all area even inside instance,I try to fix this in channel.cpp,but I think it's too hard for me.
